src/test/java/com/google/api/codegen/testdata/python_main_library.baseline

```diff
@@ -208,24 +208,25 @@ class LibraryServiceApi(object):
             scopes = self._ALL_SCOPES
         if client_config is None:
             client_config = {}
+        goog_api_client = '{}/{} {} gax/{} python/{}'.format(
+            app_name,
+            app_version,
+            self._CODE_GEN_NAME_VERSION,
+            self._GAX_VERSION,
+            platform.python_version()
+        )
+        metadata = [('x-goog-api-client', goog_api_client)]
         default_client_config = json.loads(pkg_resources.resource_string(
             __name__, 'library_service_client_config.json'))
-        self._defaults = api_callable.construct_settings(
+        defaults = api_callable.construct_settings(
             'google.example.library.v1.LibraryService',
             default_client_config,
             client_config,
             config.STATUS_CODE_NAMES,
             timeout,
+            metadata=metadata,
```

Comment:
As we discussed in the PR of gax-python, this is going to be `kwargs={'metadata': metadata}`